app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -132,6 +134,16 @@ class HomePresenter(
         }
     }
 
+    override fun onSendClicked() {
+        fallbackUsdcTokenForBuy?.let { token ->
+            if (state.tokens.isEmpty()) {
+                view?.showSendNoTokens(token)
+            } else {
+                view?.navigateToSend(newSendEnabledFeatureToggle.isFeatureEnabled)
+            }
+        }
```

Comment:
This is wrong. 
First, you should check `newSendEnabledFeatureToggle.isFeatureEnabled`. 
If new send is **disabled**, then you just open old send fragment
if new send is **enabled**, then you check if you user has tokens or not.

What is `fallbackUsdcTokenForBuy`? I feel like you don't need it